src/Libraries/CoreNodes/Color.cs

```diff
@@ -19,7 +19,7 @@ public class Color
         /// <summary>
         ///     Find the red component of a color, 0 to 255.
         /// </summary>
-        /// <returns name="red">int between 0 and 255 inclusive.</returns>
+        /// <returns name="integer">Red value for RGB color model, int between 0 and 255 inclusive.</returns>
```

Comment:
I think we should just say `int`. A lot of nodes just have `int`.

Reply:
addressed in b310d8f